src/com/facebook/buck/ide/intellij/IjProjectTemplateDataPreparer.java

```diff
@@ -503,6 +513,41 @@ private void addAndroidResourcePaths(
     }
   }
 
+  private void addAndroidCompilerOutputPath(
+      Map<String, Object> androidProperties,
+      IjModule module,
+      Path moduleBasePath) {
+    // This is the fallback default in case we can't find a valid build target name
+    String targetName = "src_release";
```

Comment:
I do not think we should have a default. There can be resource only modules that have no sources which will not have a compiler output path and they will be fine inheriting from the root project compiler output path